### DISCUSSION TITLE
HmIP-FALMOT-C12 Ch. 13-16

### DISCUSSION THREAD

**Discussion by hosswald** [SukramJ/aiohomematic#1510] [Ideas]
Currently, only Channel 1-12 are visible in Home Assistant.
It would be useful to have Channel 13-16 as well:
* Ch. 13: Wärmebedarf Boiler (`CLIMATECONTROL_HEAT_DEMAND_BOILER_TRANSMITTER`)
* Ch. 14: Wärmebedarf Pumpe (`CLIMATECONTROL_HEAT_DEMAND_PUMP_TRANSMITTER`)
* Ch. 15: Luftfeuchtigkeitssteuerung (`CLIMATECONTROL_DEHUMIDIFIER_TRANSMITER`)
* Ch. 16: MIOB Umschaltung Heizen/Kühlen (`CLIMATECONTROL_INPUT_RECEIVER`)

**Comment by SukramJ** [comment]
Please use service [export_device_definition](https://github.com/danielperna84/custom_homematic#homematicip_localexport_device_definition) and attch device dedefinitions here, or create a pullrequest at [pydevccu](https://github.com/danielperna84/pydevccu)

**Comment by hosswald** [comment]
@SukramJ 
Exports below. If there is nobody else wants to take care of this, I can try to give it a shot.

[homematicip_local/export_device_descriptions/HmIP-FALMOT-C12.json](https://github.com/danielperna84/hahomematic/files/14995539/device_descriptions_HmIP-FALMOT-C12.json)

[homematicip_local/export_paramset_descriptions/HmIP-FALMOT-C12.json](https://github.com/danielperna84/hahomematic/files/14995541/paramset_descriptions_HmIP-FALMOT-C12.json)

**Reply by SukramJ** [reply]
Bitte häng die Dateien an.

**Reply by hosswald** [reply]
Erledigt.

**Reply by SukramJ** [reply]
Weder aus den Geätebeschreibungen (paramset_description) noch aus der [technischen Dokumentation](https://homematic-ip.com/sites/default/files/downloads/hmip-device-documentation.pdf) ist ersichtlich, das das Gerät diese Datenpunkte hat.

**Reply by hosswald** [reply]
@SukramJ die Kanäle sind sowohl in der technischen Dokumentation aufgeführt (Abschnitte 117.2, 60.2, 61.2 und 62.2 für V1/V2/V3/V4):

0.    MAINTENANCE
1.    CLIMATECONTROL_FLOOR_TRANSCEIVER
2.    CLIMATECONTROL_FLOOR_TRANSCEIVER
3.    CLIMATECONTROL_FLOOR_TRANSCEIVER
4.    CLIMATECONTROL_FLOOR_TRANSCEIVER
5.    CLIMATECONTROL_FLOOR_TRANSCEIVER
6.    CLIMATECONTROL_FLOOR_TRANSCEIVER
7.    CLIMATECONTROL_FLOOR_TRANSCEIVER
8.    CLIMATECONTROL_FLOOR_TRANSCEIVER
9.    CLIMATECONTROL_FLOOR_TRANSCEIVER
10.    CLIMATECONTROL_FLOOR_TRANSCEIVER
11.    CLIMATECONTROL_FLOOR_TRANSCEIVER
12.    CLIMATECONTROL_FLOOR_TRANSCEIVER
13.    **CLIMATECONTROL_HEAT_DEMAND_BOILER_TRANSMITTER**
14.    **CLIMATECONTROL_HEAT_DEMAND_PUMP_TRANSMITTER**
15.    **CLIMATECONTROL_DEHUMIDIFIER_TRANSMITER**
16.    **CLIMATECONTROL_INPUT_RECEIVER**

sowie in paramset_descriptions unter den keys "VCU9245535:13" bis "VCU9245535:16" .

**Reply by SukramJ** [reply]
Wenn die Kanäle unter VALUES keine weiteren Parameter enthalten, dann sind die für mich nutzlos.

**Reply by hosswald** [reply]
Heißt das, der Wert der Boolean-Variable lässt sich nicht auslesen?
Tatsächlich wird er auch in RaspberryMatic nirgends angezeigt.
Ich habe dort allerdings eine funktionierende Direktverknüpfung zwischen HmIP-FALMOT-C12 Kanal 14 und einem HmIP-FSM Schaltaktor (Kanal 2).
Reine Neugier: Ist es üblich, dass Kanäle von HM(ip)-Geräten sich nicht auslesen lassen, sondern nur CCU-intern in Direktverknüpfungen oder Scripten nutzen lassen?

**Reply by SukramJ** [reply]
> Heißt das, der Wert der Boolean-Variable lässt sich nicht auslesen?

Welche?

> Tatsächlich wird er auch in RaspberryMatic nirgends angezeigt.

Und dann machst Du hier ein Ticket auf?

> Reine Neugier: Ist es üblich, dass Kanäle von HM(ip)-Geräten sich nicht auslesen lassen, sondern nur CCU-intern in Direktverknüpfungen oder Scripten nutzen lassen?

Scheinbar

**Reply by hosswald** [reply]
@SukramJ ich habe das für eine Limitierung des RaspberryMatic-Frontends gehalten und bin nicht davon ausgegangen, dass sich existierende Kanäle nicht auslesen lassen. Ich bin kein Contributor von hahomematic. Jetzt weiß ich es. Danke für die Aufklärung.
